include/seqan3/search/fm_index/bi_fm_index_cursor.hpp

```diff
@@ -82,6 +82,8 @@ class bi_fm_index_cursor
     using sdsl_char_type = typename index_type::sdsl_char_type;
     //!\brief Type of the alphabet size in the underlying SDSL index.
     using sdsl_sigma_type = typename index_type::sdsl_sigma_type;
+    //!\brief Alphabet type of the index.
+    using index_char_type = typename index_t::char_type;
```

Comment:
Why is not called `index_alphabet_type` then? The naming is misleading.

Reply:
Indeed, but then I would propose to rename it from char to alphabet everywhere in the search / fm_index module where it refers to an actual alphabet and not some kind of char/wchar , etc.

Reply:
Then I would leave it like this to be consistent with the current index design and reevaluate changing it everywhere in another PR